server/config/fusor.yaml

```diff
@@ -228,7 +240,7 @@
            :name: "OpenShift"
            :content: :openshift
            :subscription_descriptions:
-             - "Red Hat Cloud Infrastructure"
+             - "Red Hat OpenShift Enterprise"
```

Comment:
This doesn't look right.

I would think the subscription a customer is using is "Red Hat Cloud Infrastructure"

Why did you change it to "Red Hat OpenShift Enterprise", that doesn't follow the pattern of other entries.